go/consensus/tendermint/abci/mux.go

```diff
@@ -80,9 +95,11 @@ type Application interface {
 	// and Query calls and accessing genesis state.
```

Comment:
comment above still mentions `for de-multiplexing SetOption` which is not relevant anymore